src/types/custom.js

```diff
@@ -0,0 +1,66 @@
+import {Element} from 'chart.js';
+import {getChartRect, getRectCenterPoint} from '../helpers';
+
+export default class CustomAnnotation extends Element {
+
+  inRange(mouseX, mouseY, useFinalPosition) {
```

Comment:
It's the same of box annotation. I have created a function `inPointRange` in the `helpers`, maybe could be good to have `inBoxRange`?